turbinia/processors/mount_local.py

```diff
@@ -49,27 +49,28 @@ def GetDiskSize(source_path):
   Returns:
     int: the size of the disk in bytes.
   """
-  size = None
+  size = 0
```

Comment:
Are these changes supposed to be in this PR?  They look fine, but if you want to add them, maybe update the PR description/summary.

Reply:
Oh, I see a comment about these in the PR now.  Small nit again, but maybe update that in the PR summary?